rotkehlchen/db/snapshots.py

```diff
@@ -225,31 +229,31 @@ def import_snapshot(
         balances_snapshot_file: Path,
         location_data_snapshot_file: Path,
     ) -> Tuple[bool, str]:
-        main_currency = self.db.get_main_currency()
         balances_list = self._csv_to_dict(balances_snapshot_file)
         location_data_list = self._csv_to_dict(location_data_snapshot_file)
         # check if the headers match the type stored in the db
         has_invalid_headers = (
-            tuple(balances_list[0].keys()) != ('timestamp', 'category', 'asset_identifier', 'amount', f'{main_currency.symbol.lower()}_value') or  # noqa: E501
-            tuple(location_data_list[0].keys()) != ('timestamp', 'location', f'{main_currency.symbol.lower()}_value')  # noqa: E501
+            tuple(balances_list[0].keys()) != ('timestamp', 'category', 'asset_identifier', 'amount', 'usd_value') or  # noqa: E501
+            tuple(location_data_list[0].keys()) != ('timestamp', 'location', 'usd_value')  # noqa: E501
         )
         if has_invalid_headers:
             return False, 'csv file has invalid headers'
 
         # check if all timestamps are the same.
         balances_timestamps = [int(entry['timestamp']) for entry in balances_list]
-        location_data_timestamps = [entry['timestamp'] for entry in location_data_list]
+        location_data_timestamps = [int(entry['timestamp']) for entry in location_data_list]
         has_different_timestamps = (
             balances_timestamps.count(balances_timestamps[0]) != len(balances_timestamps) or
             location_data_timestamps.count(location_data_timestamps[0]) != len(location_data_timestamps) or  # noqa: E501
-            balances_timestamps[0] != create_timestamp(location_data_timestamps[0])
+            balances_timestamps[0] != location_data_timestamps[0]
         )
         if has_different_timestamps:
             return False, 'csv file has different timestamps'
 
         # check if the timestamp can be converted to int
         try:
             timestamp = Timestamp(int(balances_list[0]['timestamp']))
+            timestamp = Timestamp(int(location_data_list[0]['timestamp']))
```

Comment:
What are you doing here? you are redefining the variable you have already defined at line 255.

Reply:
This is just to check that the timestamp column contains a machine-readable timestamp and not datetime string.

Reply:
Look at my comment please.

You are doing:

```python
a = 1
a = 2
```

And redefining the same variable which is a logic error. Even lgtm.com found it and raised an error in the PR: https://lgtm.com/projects/g/rotki/rotki/snapshot/8227d39957941f2f58892ceba1074ca3dfb34378/files/rotkehlchen/db/snapshots.py?sort=name&dir=ASC&mode=heatmap#x922ace880620ed7c:1

You can use the `deserialize_timestamp()` function and then you should assign timestamp only from one of these two. Or use a different variable and also make sure they are equal.